### PR TITLE
Documents the transform upgrade feature

### PR DESCRIPTION
**Pull request by chamikaramj**
For more information, please see the Github issue: https://github.com/apache/beam/issues/27943

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

File: website/www/site/content/en/documentation/programming-guide.md [hunk 1]
```diff
@@ -8105,40 +8105,64 @@ class RetrieveTimingDoFn(beam.DoFn):
...
 
 * The Java [`PythonExternalTransform` API](https://github.com/apache/beam/blob/master/sdks/java/extensions/python/src/main/java/org/apache/beam/sdk/extensions/python/PythonExternalTransform.java) automatically
 starts the Transform service when a Python runtime isn't available locally, but Docker is.
 
 * The Apache Beam Python multi-language wrappers might automatically start the Transform service when you're using Java transforms, a Java language runtime isn't available locally, and Docker is available locally.
 
-To manually start a Transform service instance by using utilities provided with the Apache Beam SDKs, use the following commands.
+Beam users also have the option to [manually start](/documentation/programming-guide/#transform-service-usage-muanual) a stransform service and use that as the expansion service used by multi-language pipelines.
+
+### 15.2 Manually starting the transform service {#transform-service-usage-muanual}
```
Comment (liferoad) [inline website/www/site/content/en/documentation/programming-guide.md:8163]: 15.3?

Comment (chamikaramj) [reply website/www/site/content/en/documentation/programming-guide.md]: Done.

File: website/www/site/content/en/documentation/programming-guide.md [hunk 2]
```diff
@@ -8168,9 +8192,9 @@ This feature is currently in development.
 
 ### 15.2 Portable transforms included in the Transform service {#transform-service-included-transforms}
```
[the remark this answers is not in the document]
Comment (chamikaramj) [reply website/www/site/content/en/documentation/programming-guide.md]: Done.

**Review by chamikaramj — COMMENTED** [review]
Thanks. PTAL.

**Review by liferoad — APPROVED** [review]

**Pull request merged**